xml/containers-basics.xml

```diff
@@ -135,21 +134,18 @@
   <para>
    Both base and custom images are usually available through a repository of
    images called registry. Unless a registry is explicitly specified, &podman;
-   pulls images from the Docker Hub registry.
+   pulls images from the &dhub; registry. While you can fetch a base image manually, &podman; can do that automatically when building a custom image.
```

Comment:
One of the other advantages of `podman` over `docker` is that it can be configured to search multiple registries.

For example, on openSUSE the default `/etc/containers/registries.conf` contains the following:

```
[registries.search]
registries = ["registry.opensuse.org", "docker.io"]
```

This means that `podman` searches the openSUSE Registry for any requested tags before using Docker Hub as a fallback.

Even if we dont' do something by default in SLE yet (why not?) I think it's probably a good thing to document this functionality and get users thinking about different registries and how to use them, either by configuring `/etc/containers/registries.conf` or by including the full registry domain in all of their `docker|podman pull` calls, etc.